client/src/main/java/org/dst/client/cmd/DstCmdStarter.java

```diff
@@ -15,12 +16,30 @@
           commandHandlers = new HashMap<>();
 
   public static void main(String[] args) {
-    //TODO(jyx) Check the server is open or not
 
+    String address;
     if (args.length == 0) {
-      client = new DefaultDstClient(defaultAddress);
+      address = defaultAddress;
+    } else {
+      try {
+        address = ArgsParseUtil.covertArgsToAddress(args);
+      } catch (DstException e) {
+        System.out.println(e.getMessage());
+        return;
+      }
+    }
+
+    try {
+      client = new DefaultDstClient(address);
+    } catch (NullPointerException e) {
+      System.out.println("can't connect to the server, please check your input.");
```

Comment:
How about this：
connect failure，please check your input.

Reply:
Of course.